src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -115,7 +120,7 @@
  * </ul>
```

Comment:
You need to update the tool documentation to reflect the fact that multiple intervals are now supported. Also included any extra information the user might need to know (eg., are the intervals required to be contiguous?)

Reply:
Fixed doc - intervals need not be contiguous

Reply:
Added test for non adjancent intervals